sources/tests/docker-compose.yml

```diff
@@ -25,6 +25,9 @@ services:
       - REDIS_PASSWORD=${REDIS_PASSWORD}
       - REDIS_STANDALONE_MODE=${REDIS_STANDALONE_MODE}
       - QUEUE_PREFIX=${QUEUE_PREFIX}
+      - QUEUE_DASHBOARD_TOKEN_SECRET=${QUEUE_DASHBOARD_TOKEN_SECRET}
+      - QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS=${QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS}
```

Comment:
Same comment about local not requiring to authnticate.

Reply:
Same answer 😄 
Is there a problem with having the local authentication enabled?
The basic auth was also enabled in the local.